common/static/js/capa/src/formula_equation_preview.js

```diff
@@ -139,8 +139,7 @@ formulaEquationPreview.enable = function () {
                         if (inputData.jax) {
                             // Set the text as the latex code, and then update the MathJax.
                             MathJax.Hub.Queue(
-                                ['Text', inputData.jax, latex],
-                                ['Reprocess', inputData.jax]
+                                ['Text', inputData.jax, latex]
```

Comment:
Sounds fine as is.  I didn't know if sample code had "Text" followed by "Reprocess", but the first code I found had "Text" alone.  Thanks.